toolkit/tools/internal/timestamp/timestamp_test.go

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like `StopEventByPath` `PauseEvent` and `ResumeEvent` aren't covered here.

Reply:
(From https://github.com/microsoft/CBL-Mariner/actions/runs/5063082281)